chainer/training/extensions/computational_graph.py

```diff
@@ -10,9 +10,9 @@
 _func_style = {'shape': 'record', 'fillcolor': '#6495ED', 'style': 'filled'}
 
 
-def dump_graph(root_name, out_name='cg.dot',
-               variable_style=None, function_style=None):
-    """Returns a trainer extension to dump a computational graph.
+class dump_graph(extension.Extension):
```

Comment:
How about using a class-like name here (`DumpGraph`) and leave `dump_graph` a thin function that constructs and returns it? It may be also good to set `default_name = 'dump_graph'` as a class attribute for backward compatibility.

Reply:
Thank you, I renamed it according to naming rule of `class` and add `default_name` attribute.